gateway/conf.d/apicast.conf

```diff
@@ -43,9 +52,12 @@ location = /___http_call {
 
 location @out_of_band_authrep_action {
   internal;
-
   proxy_pass_request_headers off;
 
+  {% if jaeger_enabled %}
```

Comment:
This is impossible to do. This file has to stay plain nginx file without liquid templating because of our test suite.